src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/organization_policy_v2.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+"""
+Organizations Policy module used throughout the ADF.
+"""
+
+import glob
+import os
+import ast
+import json
+import boto3
+
+from logger import configure_logger
+from organization_policy_campaign import (
+    OrganizationPolicyApplicationCampaign,
+)
+from errors import ParameterNotFoundError
+
+
+from organizations import Organizations
+
+from organization_policy_schema import OrgPolicySchema
+
+LOGGER = configure_logger(__name__)
+REGION_DEFAULT = os.getenv("AWS_REGION")
+DEFAULT_ADF_POLICIES_DIR = "./adf-policies"
+
+
+class OrganizationPolicy:
+    adf_policies_dir: str
+
+    def __init__(self, adf_policies_dir=None):
+        self.adf_policies_dir = adf_policies_dir or DEFAULT_ADF_POLICIES_DIR
+
+    @staticmethod
+    def _find_all_legacy_policies(policy):
+        _files = list(
+            glob.iglob(
+                f"./adf-bootstrap/**/{policy}.json",
+                recursive=True,
+            )
+        )
+        return [f.replace("./adf-bootstrap", ".") for f in _files]
+
+    def _find_all_polices(self, policy):
+        _files = list(
+            glob.iglob(
+                f"{self.adf_policies_dir}/{policy}/*.json",
+                recursive=True,
+            )
+        )
+        return [f.replace(f"{self.adf_policies_dir}", ".") for f in _files]
+
+    @staticmethod
+    def _trim_scp_file_name(policy):
+        return policy[1:][:-8] if policy[1:][:-8] == "/" else policy[2:][:-9]
```

Comment:
Can you add comments here, explaining why 1 and -8 were chosen?
Plus what is the other use case, with 2 and -9?

The function is called trim, while it seems to only return a "/" in the first use case.
Is this correct?